pkg/resource/certificate_authority/custom_delta.go

```diff
@@ -16,37 +16,30 @@ package certificate_authority
 import (
 	svcapitypes "github.com/aws-controllers-k8s/acmpca-controller/apis/v1alpha1"
 	ackcompare "github.com/aws-controllers-k8s/runtime/pkg/compare"
-	"github.com/aws/aws-sdk-go/aws"
 )
 
 func customSetDefaults(
-	a *resource,
-	b *resource,
+	desired *resource,
+	latest *resource,
 ) {
-	// Default value of UsageMode is GENERAL_PURPOSE
-	defaultUsageMode := aws.String("GENERAL_PURPOSE")
-
-	if ackcompare.IsNil(a.ko.Spec.UsageMode) && ackcompare.IsNotNil(b.ko.Spec.UsageMode) {
-		a.ko.Spec.UsageMode = defaultUsageMode
+	if ackcompare.IsNil(desired.ko.Spec.UsageMode) && ackcompare.IsNotNil(latest.ko.Spec.UsageMode) {
+		desired.ko.Spec.UsageMode = latest.ko.Spec.UsageMode
```

Comment:
I think the naming is flipped.

You want to set latest/current to the desired state.